subsys/net/lib/zperf/zperf_udp_uploader.c

```diff
@@ -222,6 +300,14 @@ static int udp_upload(int sock, int port,
 			delay = 0U; /* delay should never be negative */
 		}
 
+		/* add clock compensate to packet delay when clock accuracy is lower than 1KHz */
+#ifdef ZPERF_UDP_UPLOAD_CLOCK_COMPENSATE
+		compensate_delay = cal_compensate_delay(loop_time, (int)packet_duration_us,
+			(int)delay, &period_start, &period, &compensate_ticks);
```

Comment:
Please fix the argument alignment:
```suggestion
		compensate_delay = cal_compensate_delay(loop_time, (int)packet_duration_us,
						        (int)delay, &period_start, &period, 
						        &compensate_ticks);
```